core/commands/refs.go

```diff
@@ -52,7 +52,9 @@ with the following format:
 
   <link base58 hash>
 
-NOTE: List all references recursively by using the flag '-r'.
+List all references recursively by using the flag '-r'.
+
+NOTE: Kubo will try to fetch the blocks of the passed path if they can't be found in the local store.
```

Comment:
Literally all commands that read blocks do this when you are in online mode.
I don't see why this one in particular should be precised, is this particularly unexpected with this command ?

Reply:
Developers who are starting out with the js stack don't really know what preload nodes are. Once we explain that it's just nodes exposing the refs endpoint, it's still not clear that the refs endpoint tries to fetch those blocks.

As far as I know we use this content to generate the docs and to me this is an absolute must.

Reply:
It isn't that clear. It would make sense if `refs -r` did not bother fetching block with a CID format of RAW, because they are always leaf nodes. I found this issue because I was wondering about that.